TravelPoints-UI/src/app/services/tourist-attraction.service.ts

```diff
@@ -27,4 +27,12 @@ export class TouristAttractionService {
         .set('Authorization', `Bearer ${token}`)
       return this.httpClient.post<TouristAttraction>(this.baseURL + "/create", touristAttraction, {headers: header});
     }
+
+  deleteTouristAttractionById(id: any): Observable<Response> {
+    let token = localStorage.getItem("token")
+    let header = new HttpHeaders()
+      .set('Content-Type', 'application/json')
+      .set('Authorization', `Bearer ${token}`)
+    return this.httpClient.delete<Response>(this.baseURL + "/delete/" + id, {headers: header});
+  }
```

Comment:
S: instead of adding the headers on every request, you can create an interceptor does that before sending any request to the backend.

Use this for reference: https://stackoverflow.com/questions/54627376/set-token-to-headers-in-angular